examples/utils/example-utils/src/containerViewRuntimeFactory.ts

```diff
@@ -44,9 +27,27 @@ export class ContainerViewRuntimeFactory<T> extends BaseContainerRuntimeFactory
 	) {
 		// We'll use a MountableView so webpack-fluid-loader can display us,
 		// and add our default view request handler.
-		super(new Map([[dataStoreFactory.type, Promise.resolve(dataStoreFactory)]]), undefined, [
-			mountableViewRequestHandler(MountableView, [makeViewRequestHandler(viewCallback)]),
-		]);
+		super(
+			new Map([[dataStoreFactory.type, Promise.resolve(dataStoreFactory)]]),
+			undefined,
+			undefined,
+			undefined,
+			async (containerRuntime: IContainerRuntime) => {
+				// ISSUE: IContainerRuntime doesn't have methods that expose data stores as IDataStore or
+				// IFluidDataStoreChannel, which expose entryPoint. getRootDataStore returns an IFluidRouter.
+				const dataStore: IFluidDataStoreChannel = (await containerRuntime.getRootDataStore(
+					dataStoreId,
+				)) as IFluidDataStoreChannel;
+
+				// TODO: better type discovery
+				const fluidObject: T = (await dataStore.entryPoint?.get()) as T;
+				if (fluidObject === undefined) {
+					throw new Error("DataStore did not set its EntryPoint");
+				}
+
+				return new MountableView(viewCallback(fluidObject));
```

Comment:
One thing to note here is that the current implementation can return one of two things (due to mountableViewRequestHandler):
1. The data object itself, if the request is made without the mountableView header
2. The data object wrapped in a MountableViewClass, if made with the mountableView header

As written here it can only return the mountable view though.  Over time I've been removing the reliance on request from a number of demos so losing the ability to get the data object directly might not actually affect anything anymore?

In addition to testing these demos directly, probably just also test data-object-grid which hosts codemirror and prosemirror in particular, make sure the "open in new window" button works, etc.